source/includes/_ais.md

```diff
@@ -99,6 +99,17 @@ To access a customer's Pots, use the Pots endpoint described further on in this
 Note that the fields we return as part of the response depend on whether your consent has the  `ReadAccountsBasic`
 or `ReadAccountsDetail` permission. In the former case, we will omit the account scheme data such as account number and sort code or IBAN.
 
+### Business Account - Company types
+
+Business account has company type, which is shown in the `Description` field:
+
+| Company type        | Description field              |
```

Comment:
Maybe we should specifying that `Description` field should not be treated as fixed enum, and can/will be changed in the future? (Or maybe, to avoid confusion, better to remove this table at all, and just mention that company type is shown in description field?)

Reply:
I think the table is useful because it shows what values the business account type can be. However I think I would choose to show only `Sole Trader` and `All other business accounts`